bin/node/src/cli.rs

```diff
@@ -69,14 +70,10 @@ pub enum Subcommand {
     #[command(subcommand)]
     Key(sc_cli::KeySubcommand),
 
-    /// Populate authorities keystore and generate chainspec in JSON format (printed to stdout)
-    /// Use `--raw` to produce the so called raw chainspec
+    /// Generates keystore (libp2p key and session keys), and generates chainspec to stdout
     BootstrapChain(BootstrapChainCmd),
 
-    /// Generate and print to stdout keys for a single node
-    BootstrapNode(BootstrapNodeCmd),
```

Comment:
This is not good imho, incorporating boostrapnode into bootsrapchain is a no go for me.